truss/cli/remote_cli.py

```diff
@@ -1,3 +1,5 @@
+from typing import Dict, Optional
```

Comment:
nit: since we're on Python 3.9+, we can use the built-in `dict` type instead of `typing.Dict` for annotations